packages/webcert/src/components/error/ErrorCopyText.tsx

```diff
@@ -46,7 +46,7 @@ const ErrorCopyText: React.FC<Props> = ({ errorId }) => {
   return (
     <Root className={'iu-pt-400'}>
       <p>
-        <strong>Fel-id:</strong> {errorId} <StyledIcon src={list} alt="Logo Kopiera text" onClick={handleCopyClick} />
+        <strong>Fel-id:</strong> {errorId} <StyledIcon src={list} alt="Kopiera text" onClick={handleCopyClick} />
```

Comment:
From an accessibility standpoint we should avoid using onClick on images, if it's a button it should semantically be a button.